charts/connect/templates/connect-deployment.yaml

```diff
@@ -14,6 +14,7 @@ metadata:
   {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
+  replicas: {{ .Values.connect.replicas }}
```

Comment:
I'd like to get some explanation on why this line was moved. I want to make sure we understand this change.

Reply:
Ok, I'll add.

But in general. The replicas should be specified at the top level `spec` according to the [documentation](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/#replicas)

Reply:
Gotcha. This explains it. Thank you.

Reply:
I think there's a misunderstanding here. I've realized that by looking deeper into the code.
While you are correct about `spec` being mentioned at the top level, in this case we use a template that has a spec. I think that the template's spec is the place where the replicaset should be mentioned rather than where it is now (with your change).

Reply:
Replicas can't be set within `.spec.template.spec` as it will throw the error when you try to apply deployment.
```
 ValidationError(Deployment.spec.template.spec): unknown field "replicas" in io.k8s.api.core.v1.PodSpec
```
I think everything is fine here. And it works as expected.

Reply:
Understood. Thank you for looking into this.